Poker.BE/Poker.BE.Domain/Security/UserManager.cs

```diff
@@ -75,8 +75,10 @@ public bool EditProfile (string oldUserName, string newUserName, string newPassw
 
             if (CheckExistingUser(newUserName) || !CheckPasswordValidity(newPassword))  //Check new username and password validation
```

Comment:
🤕 ummm... this is good for now, later we can improve it by Authentication & Authorization procedures of WEB.API .NET.
[you can find info here](https://docs.microsoft.com/en-us/aspnet/web-api/overview/security/authentication-and-authorization-in-aspnet-web-api)